packages/react-icons/package.json

```diff
@@ -16,7 +16,7 @@
     "cleanSvg": "rm -rf ./intermediate",
     "copy": "node ../../importer/generate.js --source=../../assets --dest=./intermediate --extension=svg --target=react",
     "copy:font-files": "cpy './src/utils/fonts/*.{ttf,woff,woff2,json}' ./lib/utils/fonts/. && cpy './src/utils/fonts/*.{ttf,woff,woff2,json}' ./lib-cjs/utils/fonts/.",
-    "convert:svg": "node convert.js --source=./intermediate --dest=./src",
+    "convert:svg": "node convert.js --source=./intermediate --dest=./src --filter=../../importer/rtl.txt",
```

Comment:
The `rtl.txt` file is [here](https://github.com/microsoft/fluentui-system-icons/blob/main/importer/rtl.txt).

I am wary about creating a new file for this because this file was already created by the designers to track icons that should be flipped in rtl

Reply:
We need to track down with the Android folks (that's the only platform previously using the rtl.txt file) what is/isn't there. RTL wasn't kept up with, for the most part, as other platforms didn't need/use it.
We should also collaborate with the Apple folks on that, as they are the platform that has the most RTL/LTR support in, and it's supported by the platform itself, but we need to make sure everything has equivalency.